nion/swift/ImageCanvasItem.py

```diff
@@ -240,12 +240,20 @@ class ScaleMarkerCanvasItemComposer(CanvasItem.BaseComposer):
     def __init__(self, canvas_item: CanvasItem.AbstractCanvasItem, layout_sizing: CanvasItem.Sizing,
                  cache: CanvasItem.ComposerCache, dimensional_calibration: typing.Optional[Calibration.Calibration],
                  info_text: str, screen_pixel_per_image_pixel: typing.Optional[float],
-                 get_font_metrics_fn: typing.Callable[[str, str], UISettings.FontMetrics]) -> None:
+                 get_font_metrics_fn: typing.Callable[[str, str], UISettings.FontMetrics],
+                 scale_marker_position: str| None,
+                 is_background_enabled: bool,
+                 background_fill_color: str | None,
```

Comment:
I think you only need `background_fill_color` here since it is optional. If it is `None` it won't draw. `is_background_enabled` would be redundant.

There is a larger issue of how we specify that something is inheriting from preferences vs. setting its value to a specific color or clearing the inherited value, but the extra `is_background_enabled` doesn't cover that case well either. So I think it is redundant.